### ISSUE TITLE
Copy Document action

### ISSUE THREAD

**Issue by dgrumieaux** [tsgrp/hpi#763]
In HPI 1.x, the copy document feature was available to take a document and copy it. This enhancement will allow a user to do the same, potentially allowing a user to define where they want to place the new copy.

**Comment by gsteimer** [comment]
Related? #470

**Comment by dgrumieaux** [comment]
Definitely related, but #470 will be far more complex.

**Comment by gsteimer** [comment]
Right - I was just wondering if #470 could piggy back off of whatever we do here...

**Comment by dgrumieaux** [comment]
Yes, I think that's a good idea. Maybe a chained action or using the same backend code to at least take care of the basic copy functions.
